seshdash/data/db/influx.py

```diff
@@ -315,6 +313,35 @@ def get_measurement_range(self, measurement_name, start, end, site=None, databas
   
         return results
 
+    def get_measurement_range_bucket(self, measurement_name, start, end, group_by, site=None, database=None):
+        """
+        Returns the measurement points for a given range grouped by a given time
+
+        @param measurement_name - name of the measurement
+        @param start - The date to start from (type python datetime)
+        @param end - The time to stop at ( type python datetime)
```

Comment:
params for site and database are missing

database should be the database defined in configs by default.
something  like.
```
if not database:
   database = settings.INFLUX_DATABASE
```